libs/brig-types/src/Brig/Types/User/Auth.hs

```diff
@@ -43,10 +46,15 @@ newtype LoginCodeTimeout = LoginCodeTimeout
     { fromLoginCodeTimeout :: Timeout }
     deriving (Eq, Show)
 
+-- | Different kinds of logins.
 data Login
     = PasswordLogin !LoginId !PlainTextPassword !(Maybe CookieLabel)
     | SmsLogin !Phone !LoginCode !(Maybe CookieLabel)
 
+-- | A special kind of login that is only used for an internal endpoint.
+data BackdoorLogin
+    = BackdoorLogin !LoginId !(Maybe CookieLabel)
```

Comment:
`UserId` would be enough here.

Reply:
Are you saying that `CookieLabel` is not needed for your purposes, or is that it is actually bad to have here? If it's just the former then I would rather leave it in place.

Reply:
sorry, i meant: `LoginId` is too much, `UserId` would be better instead.  emails and phone numbers should not be used to identify users on this end-point for now.  we certainly need the cookie label.